### PR TITLE
Add UT for config:isCGroupV2

### PR DESCRIPTION
**Pull request by jichenjc**
will wait for merge this then follow up with https://github.com/sustainable-computing-io/kepler/issues/179

### CODE REVIEW

File: pkg/config/config_test.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		file, err := createTempFile("")
+		Expect(err).NotTo(HaveOccurred())
+		defer os.Remove(file)
+		tempfile = file
+
+		Expect(true).To(Equal(isCGroupV2(mockc)))
+
+		tempfile = "/tmp/do_not_exist"
+		Expect(false).To(Equal(isCGroupV2(mockc)))
+	})
```
Comment (SamYuan1990) [inline pkg/config/config_test.go:65]: @jichenjc , do you think we need add a test to cover `getKernelVersion` as #179 reports for `getKernelVersion`?

Comment (jichenjc) [reply pkg/config/config_test.go:65]: yes, I plan to add UT in this PR and modify code+UT for that issue with follow up PR

File: pkg/config/config_test.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+var _ = Describe("Test Configuration", func() {
+	It("Test cgroup version", func() {
+		file, err := createTempFile("")
+		Expect(err).NotTo(HaveOccurred())
+		defer os.Remove(file)
+		tempfile = file
+
+		Expect(true).To(Equal(isCGroupV2(mockc)))
+
+		tempfile = "/tmp/do_not_exist"
```
Comment (rootfs) [inline pkg/config/config_test.go:63]: a bit paranoid but what if this file does exist in the test environment?

File: pkg/config/config.go [hunk 3]
```diff
@@ -32,6 +32,16 @@ const (
 	cGroupV2Path = "/sys/fs/cgroup/cgroup.controllers"
 )
 
+type Client interface {
+	getSysInfo() ([]byte, error)
+	getCgroupV2file() string
```
Comment (rootfs) [inline pkg/config/config.go:37]: nit: `getCgroupV2File`

**Comment by rootfs** [comment]
@jichenjc can you sign your commits? https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

File: pkg/config/config_test.go (hunk 2 above)
Comment (jichenjc) [reply pkg/config/config_test.go]: updated with a more `unlikely name` :)

**Pull request merged**